src/faebryk/library/NetTie.py

```diff
@@ -0,0 +1,51 @@
+# This file is part of the faebryk project
+# SPDX-License-Identifier: MIT
+
+import logging
+from enum import Enum
+
+import faebryk.library._F as F
+from faebryk.core.module import Module
+from faebryk.core.moduleinterface import ModuleInterface
+from faebryk.library.Electrical import Electrical
+from faebryk.library.KicadFootprint import KicadFootprint
+from faebryk.libs.picker.picker import has_part_picked_remove
+
+logger = logging.getLogger(__name__)
+
+
+class NetTie[T: ModuleInterface](Module):
+    class Size(float, Enum):
+        _2_0MM = 2.0
+        _0_5MM = 0.5
+
+    unnamed: list[T]
+
+    def __init__(
+        self,
+        width: Size,
+        interface_type: type[T] = Electrical,
```

Comment:
What is a net tie for non-electrical?

Reply:
We should make a tree (maybe as a build artifact) of the types. It's non-obvious that `Electrical` is indeed the baseclass of everything a switch might be